spring-pulsar-test/src/main/java/org/springframework/pulsar/test/support/PulsarConsumerTestUtil.java

```diff
@@ -85,6 +122,12 @@ public ConditionsSpec<T> awaitAtMost(Duration timeout) {
 
 	@Override
 	public ConditionsSpec<T> until(ConsumedMessagesCondition<T> condition) {
+		if (calledUntil) {
```

Comment:
Do we really need to introduce a new variable? Is there a reason why we can't rely on `this.condition` being not being `null` to verify if `until` has been called before?

Reply:
I believe we do since we do not have to specify a condition (e.g. `PCTUT.whenConditionIsNotSpecifiedMessagesAreConsumedUntilAwaitDuration`). 

I would like to suggest to rename the var to `untilMethodAlreadyCalled` as the word `until` in general is a confusing one that makes my brain pause when reading the code "until something happens". It seems nit picky but including the "method" in there helps readability for me anyways.

Reply:
`untilMethodAlreadyCalled` is better than `calledUntil` imo